tests/ignite/contrib/handlers/test_visdom_logger.py

```diff
@@ -938,8 +938,6 @@ def update_fn(engine, batch):
 def no_site_packages():
     import sys
 
-    import visdom
-
     plx_module = sys.modules["visdom"]
```

Comment:
I think we should keep `import visdom` as otherwise it wont appear in sys modules. Maybe, same for other `no_site_packages` fixtures.

```suggestion
    visdom_module = sys.modules["visdom"]
```

Reply:
Test passed locally tho.

Reply:
I support that they can pass if there is a test that imports visdom before the test which calls `no_site_packages`... I think we should keep the import for consistency.